### PR TITLE
Explicitly callout the prerequisites of Ingress on GCE/GKE.

### PR DESCRIPTION
**Pull request by bprashanth**
The same things trip people up time and again. Unfortunately the version of GLBC we shipped with in 1.2 doesn't create the firewall rule because of a last minute bug I found in the k8s gce cloudprovider module. Adding some clarity around this. 

@kubernetes/goog-cluster

### CODE REVIEW

**Comment by bprashanth** [comment]
@freehan for tech review

File: docs/user-guide/ingress.md [hunk 1]
```diff
@@ -39,12 +39,28 @@ It can be configured to give services externally-reachable urls, load balance tr
...
-* The resource currently does not support HTTPS, but will do so before it leaves beta.
+On GCE/GKE there should be a [L7 cluster addon](https://github.com/kubernetes/contrib/blob/master/ingress/controllers/gce/README.md), deployed into the `kube-system` namespace:
+
+```shell
+$ kubectl get pods --namespace=kube-system -l name=glbc
+NAME                            READY     STATUS    RESTARTS   AGE
+l7-lb-controller-v0.6.0-chnan   2/2       Running   0          1d
+```
+
+make sure you review the [beta limitations](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/gce/BETA_LIMITATIONS.md) of this controller. In particular, you need to create a single firewall-rule on your cloudprovider, to allow health checks. On GKE this would be:
```
Comment (thockin) [inline docs/user-guide/ingress.md]: s/make/Make

Comment (thockin) [inline docs/user-guide/ingress.md]: Is this a permanent need, or will the glbc eventually support doing this?

Comment (bprashanth) [inline docs/user-guide/ingress.md]: there was a bug I found in the last minute that required a godep rev, so I punted on it for the release. I'm planning to fix it in the next minor release.

Comment (bprashanth) [inline docs/user-guide/ingress.md]: But the firewall precaution will still be required for other (non-gce) ingress controllers, so I'll probably remove the specific example but leave the text about firewall rules in there

**Comment by bprashanth** [comment]
repushed

**Comment by thockin** [comment]
Looks OK to me - I'll let @freehan final review.  Thanks!

**Comment by freehan** [comment]
LGTM

**Comment by bprashanth** [comment]
Thanks

**Comment by bprashanth** [comment]
Hmm, apparently i can't merge. @johndmulhausen merge please?

**Comment by johndmulhausen** [comment]
Thanks; go ahead and use Tech Review LGTM so I see that all is well next time :) @thockin @freehan

**Pull request merged**